cosmic-core/api/src/main/java/com/cloud/api/command/user/network/CreateNetworkCmd.java

```diff
@@ -284,6 +288,13 @@ public String getDns2() {
         return dns2;
     }
 
+    public String getIpExclusionList() {
+        if(StringUtils.isEmpty(ipExclusionList)){
```

Comment:
Not a big deal but it does help readability when we put some spaces around the parentheses: `if (StringUtils.isEmpty(ipExclusionList)) {`
😃